docs/code-style.md

```diff
@@ -356,8 +356,8 @@ are partial functions.
 
 As usual, separate type classes with `, ` (comma and a space).
 
-If you are using GHC-8.2.2 or higher you *should* use
-[`-XDerivingStrategies`](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/glasgow_exts.html#deriving-strategies)
+If you are using GHC-8.2.1 or higher you *should* use
+[`-XDerivingStrategies`](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/deriving_strategies.html)
```

Comment:
I don't think linking to the dev-version on gitlab instance is a good idea. Let's rather use a more specific link, e.g.

```suggestion
[`-XDerivingStrategies`](https://downloads.haskell.org/ghc/latest/docs/users_guide/exts/deriving_strategies.html)
```

Or we could maybe even pin the version

```suggestion
[`-XDerivingStrategies`](https://downloads.haskell.org/ghc/9.4.1/docs/users_guide/exts/deriving_strategies.html)
```

Reply:
Gone for the `latest` version. Compiler fixed version imo only makes sense if we update those links everytime we switch the "preferred" compiler version...

Reply:
Docs for deriving strategies shouldn't really change (much) regardless of what compiler version we use ourselves, so there'd be no real reason to update these links. And there's a chance `latest` will eventually move the page somewhere and the link will break. But whatever, we'll cross that bridge when we get there, I guess.